### PR TITLE
make att.namespaceable sensible

### PR DESCRIPTION
**Pull request by sydb**
Update wording of description & add remarks per #2395.

This does not (IMHO) require either 3 reviews or 3 assignees — as soon as 1 reviewer gives OK, one of the assignees should merge (presuming they agree).

### CODE REVIEW

File: P5/Source/Specs/att.namespaceable.xml [hunk 1]
```diff
@@ -20,18 +19,29 @@ $Id$
...
       <defaultVal>http://www.tei-c.org/ns/1.0</defaultVal>
+      <remarks xml:lang="en" versionDate="2023-02-07">
+        <p>When specified on an <gi>elementSpec</gi>, this attribute
+        specifies the namespace of the element being added, deleted,
+        replaced, or changed. When specified on a <gi>schemaSpec</gi>,
+        this attribute specifies the namespace of every element that
+        is added, deleted, replaced, or changed by
+        <gi>elementSpec</gi> children of the <gi>schemaSpec</gi>, or
+        by <gi>elementSpec</gi>s referred to by <gi>specGrpRef</gi>
+        children fo the <gi>schemSpec</gi>, <emph>unless</emph>
```
Comment (joeytakeda) [inline P5/Source/Specs/att.namespaceable.xml]: teeny typo: "fo" --> "of" :-)

**Review by joeytakeda — CHANGES_REQUESTED** [review]
With the exception of a tiny typo, this looks good to me

File: P5/Source/Specs/att.namespaceable.xml (hunk 1 above)
Comment (sydb) [reply P5/Source/Specs/att.namespaceable.xml]: Good catch, thanks!

**Review by martindholmes — APPROVED** [review]
This looks good to me. No problems with merging.

**Pull request merged**